spec/controllers/hunters_controller_spec.rb

```diff
@@ -12,7 +12,7 @@
         name: 'Ruudii',
         playbook_id: create(:playbook).id,
         harm: 0,
-        luck: 0,
+        luck: 7,
```

Comment:
Setting :luck and :harm to nil should result in valid attributes. Your changes to the controller allow us to pass nil values and still create a hunter. That'll test the changes you made.

Reply:
Added a test with it's own context for this to avoid chaging the `valid_attributes` which will effect other tests 😄